pmda/parallel.py

```diff
@@ -352,8 +359,12 @@ def _dask_helper(self, start, stop, step, indices, top, traj):
         res = []
         times_io = []
         times_compute = []
-        for i in range(start, stop, step):
+        assert bslice.stop is not None, \
```

Comment:
That is true in principle but it is not really documented that this is our internal API. I'd rather have some defensive programming here, in case someone changes things in a year's time. And I rather put in an assert than a comment.

Reply:
> We have the test-suite to establish that trust.
Point taken, I'll rely on the tests and leave a comment in.